inspire_dojson/hepnames/rules.py

```diff
@@ -224,6 +225,8 @@ def name2marc(self, key, value):
 
     if 'name_variants' in value:
         self['400'] = [{'a': el} for el in value['name_variants']]
+    if 'previous_names' in value:
+        self['667'] = {'a': ';'.join(value['previous_names'])}
```

Comment:
I wouldn't join them, I would do
```suggestion
        self['667'] = [{'a': "Previously known as {}".format(prev_name)} for prev_name in value['previous_names']]
```

Reply:
Well, if you give a look to the data we have, that format doesn't really suite well. Some example of `667__a` field are:
* `Sakurai Prize 1988 Dirac Medal 1998` http://inspirehep.net/record/1018932/export/xm
* `2009 APS Dissertation Award in Nuclear Physics` http://inspirehep.net/record/1046396/export/xm
* `Do not confuse with Acharya, Bobby Samir` http://inspirehep.net/record/1060601/export/xm

Reply:
BTW, the `;` join is there because it is specified in the US https://its.cern.ch/jira/browse/INSPIR-2156

Reply:
Final decision is that we parse the `667__a` field reading the content only if the field contains `Formerly`.